src/python/openvino_genai/py_openvino_genai.pyi

```diff
@@ -1666,12 +1666,12 @@ class Tokenizer:
         Decode a batch of tokens into a list of string prompt.
         """
     @typing.overload
-    def encode(self, prompts: list[str], add_special_tokens: bool = True) -> TokenizedInputs:
+    def encode(self, prompts: list[str], add_special_tokens: bool = True, max_length: int = 2147483647, padding_mode: str = 'truncate') -> TokenizedInputs:
```

Comment:
how to read it? 
E.g. `max_length: int = 2147483647` can not be used for ClipTokenizer 

`padding_mode: str` -> `padding_mode: PaddingMode`. 
BTW, I see that `src/python/py_tokenizer.cpp` defines Variant for types, while here we see only `str`

Reply:
I changed that. The current approach is way way simpler and clearer. Only enum is accepted. Please take a look to the latest version